### ISSUE TITLE
Original BERT-BASE / BERT-LARGE training loss across training steps

### ISSUE THREAD

**Issue by iliaschalkidis** [google-research/bert#921]
Hi @jacobdevlin-google ,

I would like to know if there is any plot reporting the original BERT-BASE and BERT-LARGE training loss across training steps (from 0 to the final 1M training steps)? 

I can't find this piece of information in any version of  your publication, neither in future publications from other groups (ALBERT, XLNET, etc.). Only Lan et al. (ALBERT) paper has something similar but for the first 100K train. steps, which is of course just a small part of the whole story (1M steps)...

Is it possible to provide something like this, as a point of reference?

Thanks!
